### ISSUE TITLE
[Bug]: Light card doesn't fit on smaller (phone) screens

### ISSUE THREAD

**Issue by xPhantomNL** [piitaya/lovelace-mushroom#405]
### Current Behavior

![image](https://user-images.githubusercontent.com/5745036/165408370-756ba529-06ef-4d6a-b24f-aee34df27356.png)

On my phone (Pixel 6), I've noticed that the light card doesn't properly fit on smaller screens (like on phones). I was able to reproduce this behaviour on my desktop by using the device view within the Chrome devtools.  

### Expected Behavior

![image](https://user-images.githubusercontent.com/5745036/165408390-b56c6254-4b39-470f-b535-106b43c58c0b.png)

I would expect that it would show just like it would on a tablet / desktop

### Steps To Reproduce

I have a lovelace view of view type "Grid (layout-card)" by using thomasloven's layout card with the following card;

```yaml
type: custom:mushroom-light-card
entity: light.livingroom_lights
hide_state: false
use_light_color: false
show_brightness_control: true
show_color_control: true
show_color_temp_control: true
```

**Comment by piitaya** [comment]
Hello !
What's your grid layout config? Looks like there is a min width on it.

**Comment by xPhantomNL** [comment]
```yaml
  - theme: Backend-selected
    title: Lights
    path: lights
    icon: fapro:lightbulb-on-duotone
    type: custom:grid-layout
    badges: []
    cards:
      - type: custom:mushroom-light-card
        entity: light.livingroom_lights
        hide_state: true
        use_light_color: false
        show_brightness_control: true
        show_color_temp_control: true
        show_color_control: true
```

This is the relevant part of the entire page, as copied from the "raw configuration editor". 

If I switch the page to any other type (default masonry or the 4 other layout-card options) it will render like I would expect on mobile.

**Comment by piitaya** [comment]
I'm unable to reproduce. What's your HA et Mushroom version?

My conf : 
```yaml
  - theme: Backend-selected
    type: custom:grid-layout
    title: aaa
    path: aaa
    badges: []
    cards:
      - type: custom:mushroom-light-card
        entity: light.yeelight_bedroom_left
        show_brightness_control: true
        show_color_control: true
        show_color_temp_control: true
        hide_state: true
```
![image](https://user-images.githubusercontent.com/5878303/165498099-cac4c8a1-ec8c-4cb2-bea0-3631fd107f56.png)

**Comment by xPhantomNL** [comment]
Environment;
Home Assistant 2022.4.6 (running a VM through Proxmox)
Mushroom 1.6.3
Layout-card 2.4.2
Chrome 100.0.4896

I've tried the exact same code in a freshly installed virtual machine (with 2022.4.7) with only the Hue integration added, and it seems to work fine with the exact same lovelace copied through the raw configuration editor. 
For now, I have fixed it by wrapping everything in a vertical-stack card and using card mod to force the width. This works fine on my Pixel 6, iPhone 12 and Galaxy Tab.

I am going to dive deeper into this issue later on, but it seems to be coming from another card and doesn't seem to be related to mushroom. Thanks for your help, though! Keep up the good work, I love it! :)
